### PR TITLE
Prepare for simple-phpunit

### PR DESCRIPTION
**Pull request by greg0ire**
Refs https://github.com/sonata-project/SonataAdminBundle/pull/4606

Don't merge until we decide to actually go with simple-phpunit

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "require-dev": {
         "jms/di-extra-bundle": "^1.7",
         "jms/translation-bundle": "^1.2",
-        "matthiasnoback/symfony-dependency-injection-test": "^0.1 || ^1.0",
+        "matthiasnoback/symfony-dependency-injection-test": "^0.7 || ^1.0",
```
Comment (core23) [inline composer.json]: Can we drop the `0.x` part?

Comment (greg0ire) [inline composer.json]: Maybe, let's try.

**Comment by OskarStark** [comment]
Travis failed for PHP 7.1 with SF 2.3

**Comment by jordisala1991** [comment]
Can you rebase @greg0ire to pick the latest changes on the repo

**Comment by jordisala1991** [comment]
> Don't merge until we decide to actually go with simple-phpunit

I just followed all the conversation through issues and PR, but I didn't found a recent votation or anything to decide. The last one was 5-1 in favour of doing this change.

Should we talk it again on slack or do another votation on github or should we move forward with this idea?

**Comment by greg0ire** [comment]
Not sure about this anymore

**Pull request closed without merging**